src/components/Methodology.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { Layout } from 'antd';
+
+class Methodology extends Component {
+  
+    render() {
+        const { Content } = Layout;
+        return (
+            <Content id="methods" style={{ background: '#fefefe', padding: '50px 0', height: '80vh' }}>
```

Comment:
You should move this color to the constants file. In the future if you ever want to change this color and its part of an overall color theme, it mights it easy to change.